portfolio/src/main/webapp/index.html

```diff
@@ -186,10 +186,21 @@ <h2 class="mb-5">Leave a comment</h2>
           </div>
             <input type="submit" value="Submit"> 
         </form>
-        <br>
-        <h3>Comment History:</h3>
-        <ul id="comment-list"></ul>
+        <hr>
 
+        <form>
+          <div class="form-group">
+            <label for="number">Change number of comments displayed:</label>
+            <select name="number-comments" id="number" onchange="loadComments()">
+              <option value="5">5</option>
+              <option value="10">10</option>
+              <option value="15">15</option>
```

Comment:
If you'd like, you can "selected" at the end of one of the options to make that the default.

Reply:
cool, thanks!